addons/argocd/disable

```diff
@@ -15,9 +15,36 @@ echo "Disabling ArgoCD"
 $HELM delete argo-cd -n $NAMESPACE_ARGOCD
 
 $KUBECTL delete customresourcedefinition applications.argoproj.io \
+                applicationsets.argoproj.io \
                 argocdextensions.argoproj.io  \
                 appprojects.argoproj.io
 
-$KUBECTL delete $KUBECTL_DELETE_ARGS namespace "$NAMESPACE_ARGOCD" > /dev/null 2>&1 || true
+PURGE=false
+# get the options
+while getopts ":p" flag; do
+  case "${flag}" in
+          p) PURGE=true
+             ;;
+          *) echo "Usage: microk8s disable argocd [-p]"
+             echo ""
+             echo "WARNING: Final deletion of \"$NAMESPACE_ARGOCD\" namespace must be enforced by: \"microk8s disable argocd -p\""
+             echo ""
+             echo "Purge only when sure, that \"$NAMESPACE_ARGOCD\" namespace is not hosting any other services from Argo stack."
+             echo ""             
+             exit 0
+             ;;
+  esac
+done
```

Comment:
Just noticed, this should be earlier in the script, before we remove the helm installation and the CRDs

Reply:
@neoaggelos right, makes sense. New commit added.